profiles/base.yaml

```diff
@@ -114,7 +114,8 @@ empty_hunting_room_messages:
 - Anyone here?
 hunting_buddies:
 stop_hunting_if_bleeding: true
-
+pre_hunt_buff: false
```

Comment:
I don't think we need a boolean here. The presence of the waggle set should be enough. What do you think?